internal/test/archive-demo/configurator.sh

```
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+UTXO_HD_WITH="mem"
+
+# Log file
+
+# Implement sponge-like command without the need for binary nor TMPDIR environment variable
+write_file() {
+    # Create temporary file
+    local tmp_file="${1}_$(tr </dev/urandom -dc A-Za-z0-9 | head -c16)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)